src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Services/Implementation/PolicyInformationPoint.cs

```diff
@@ -0,0 +1,117 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Altinn.Authorization.ABAC.Constants;
+using Altinn.Authorization.ABAC.Xacml;
+using Altinn.Platform.Authorization.Configuration;
+using Altinn.Platform.Authorization.Models;
+using Altinn.Platform.Authorization.Repositories.Interface;
+using Altinn.Platform.Authorization.Services.Interface;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Options;
+
+namespace Altinn.Platform.Authorization.Services.Implementation
+{
+    /// <summary>
+    /// The Policy Administration Point responsible for storing and modifying delegation policies
```

Comment:
```suggestion
    /// The Policy Information Point responsible for storing and modifying delegation policies
```

The rest of the comment should probably be updated as well